scripts/storagedrawers.zs

```diff
@@ -382,10 +383,10 @@ assembler.recipeBuilder()
 
 recipes.removeByRecipeName("storagedrawers:upgrade_void");
 assembler.recipeBuilder()
```

Comment:
Instead of an assembler, why don't we make this a canning machine recipe? Makes more sense to me at least. Could follow suit for all the other drawer upgrades as well.